src/wp-includes/theme.php

```diff
@@ -188,23 +191,31 @@ function get_stylesheet() {
  *
  * @since 1.5.0
  *
+ * @global string $wp_stylesheet_path Current theme stylesheet directory path.
```

Comment:
Cleverly manage global variables in the function. Awesome work, Felix.

Do we need to add/update documentation for the function so others know that starting from version 6.4, we implemented these changes? If not, then mark it as resolved.

🔢 If changed, applies elsewhere in the PR.

Reply:
Addressed in https://github.com/WordPress/wordpress-develop/pull/5160/commits/96b5f650833c15f1a97931b5a7d5a52e14201c3a